packages/atomic/cypress/e2e/result-list/result-template.cypress.ts

```diff
@@ -61,6 +66,42 @@ describe('Result Template Component', () => {
     assertContainsComponentError(ResultTemplateSelectors, true);
   });
 
+  describe('when it has both section and non-section elements', () => {
+    beforeEach(() => {
+      new TestFixture()
+        .with(
+          addResultList(
+            buildTemplateWithoutSections([
+              new Text('hello'),
+              generateComponentHTML(resultSectionTags.actions),
+            ])
+          )
+        )
+        .init();
+    });
+
+    assertConsoleWarning(true);
+  });
+
+  describe('when it has strictly section and table-element elements', () => {
+    beforeEach(() => {
+      new TestFixture()
+        .with(
+          addResultList(
+            buildTemplateWithoutSections([
+              generateComponentHTML(tableElementTagName),
+              new Text('    \n    '),
```

Comment:
HTML templates tend to contain indentation and newline characters as text nodes. We don't want to log an error for those.